### PR TITLE
January Outage Fixes

### PR DESCRIPTION
**Pull request by willronchetti**
- Multiple issues arose suddenly on the server that were fixed by the following updates (along with bringing in a newer version of goofys on the server).
- Relevant issues: https://github.com/jupyterhub/jupyterhub/issues/2213, https://github.com/jupyterhub/jupyterhub/issues/2915

### CODE REVIEW

File: jupyterhub_config.py [hunk 1]
```diff
@@ -186,11 +187,9 @@ def finalize_user_content(spawner):
 # c.DockerSpawner.extra_create_kwargs.update({ 'command': spawn_cmd })
 
 # Connect containers to this Docker network
-network_name = os.environ['DOCKER_NETWORK_NAME']
+network_name = 'bridge'
 c.DockerSpawner.use_internal_ip = True
-c.DockerSpawner.network_name = network_name
-# Pass the network name as argument to spawned containers
-c.DockerSpawner.extra_host_config = { 'network_mode': network_name }
+c.DockerSpawner.network_name = os.environ['DOCKER_NETWORK_NAME']
```
Comment (ebberg) [inline jupyterhub_config.py:192]: what does this env var evaluate to? different than bridge?

File: jupyterhub_config.py [hunk 2]
```diff
@@ -143,6 +143,7 @@ def finalize_user_content(spawner):
     - adding date_culled to the TrackingItem given by FF_TRACKING_ID
     """
     # grab this info fresh every time
+    err_output = []
```
Comment (ebberg) [inline jupyterhub_config.py:146]: what does this line do? just resets any existing errors that have been reported?

**Review by ebberg — APPROVED** [review]
looks good, although I'm curious how these fixed :)

File: jupyterhub_config.py (hunk 2 above)
Comment (willronchetti) [reply jupyterhub_config.py:146]: Missing declaration, so was a syntax error that made it through.

File: jupyterhub_config.py (hunk 1 above)
Comment (willronchetti) [reply jupyterhub_config.py:192]: See `.env`.

File: jupyterhub_config.py [hunk 3]
```diff
@@ -186,11 +187,9 @@ def finalize_user_content(spawner):
 # c.DockerSpawner.extra_create_kwargs.update({ 'command': spawn_cmd })
 
 # Connect containers to this Docker network
-network_name = os.environ['DOCKER_NETWORK_NAME']
+network_name = 'bridge'
```
Comment (netsettler) [inline jupyterhub_config.py]: Is this variable not now unused?

File: jupyterhub_config.py (hunk 1 above)
Comment (netsettler) [inline jupyterhub_config.py:192]: Should this use `network_name`, as in (untested):

```suggestion
c.DockerSpawner.network_name = os.environ.get('DOCKER_NETWORK_NAME', network_name)
```

**Review by netsettler — APPROVED** [review]
There may be an error here, so see my comment. But if you're setting the environment variable it will be harmless. Probably the choice of the name `bridge`, therefore, is not needed. I might call it `jupyterhub-network` or something like that as in in the examples you pasted me ([here](https://github.com/jupyterhub/jupyterhub/issues/2915) and [here](https://github.com/jupyterhub/jupyterhub/issues/2213)).

File: jupyterhub_config.py (hunk 3 above)
Comment (willronchetti) [reply jupyterhub_config.py]: Added a comment.

**Pull request merged**